src/scenes/home/team/boardMembers.js

```diff
@@ -27,4 +27,26 @@ const Secretary = {
   src: './images/thomas.jpg'
 };
 
-export default [Chair, Vicechair, Treasurer, Secretary];
+const board1 = {
+  name: 'Conrad Hollomon',
+  role: 'Board Director',
+  description: '',
+  src: './images/conrad.jpg'
```

Comment:
Don't do it this way, even though it was originally implemented that way. For all of the board members, import their image from src/images and then map it as the value of src like

```
import firstLastImage from 'images/firstname_lastname.png';
...
src: firstLastImage,
```

Reply:
Once you do that, you can delete images from `/public`

Also, please try to name the images clearly and uniformly, like `first_last.jpg`